core/logging/impl-file/src/jvmMain/kotlin/net/thunderbird/core/logging/file/JvmFileSystemManager.kt

```diff
@@ -0,0 +1,13 @@
+package net.thunderbird.core.logging.file
+
+import kotlinx.io.RawSink
+
+/**
+ * Android implementation of [FileSystemManager] that uses [ContentResolver] to perform file operations.
+ */
+class JvmFileSystemManager() : FileSystemManager {
+    override fun openSink(uriString: String, mode: String): RawSink? {
+        // TODO: Implementation
```

Comment:
Could you please add an issue to add the JVM implementation and add the link here.